### ISSUE TITLE
[PF4] 3.16.0 not work bug in ClipboardCopy component

### ISSUE THREAD

**Issue by aljesusg** [patternfly/patternfly-react#1871]
node_modules/@patternfly/react-core/dist/js/components/ClipboardCopy/ClipboardCopy.d.ts
(1,63): Module '"../../../../../../@types/react"' has no exported member 'HTMLDivElement'.

- Remove HTMLDivElement from import

```typescript
import { FunctionComponent, HTMLProps, MouseEvent, ReactNode, HTMLDivElement } from 'react';
```
fix
```typescript
import { FunctionComponent, HTMLProps, MouseEvent, ReactNode } from 'react';
```
- Wrong import typeUtils
```typescript
import { OneOf } from '../../typeUtils';
```
fix
```typescript
import { OneOf } from '../../helpers/typeUtils';
```
- 5,37): A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference.
```typescript
export const ClipboardCopyVariant = {
  inline: 'inline',
  expansion: 'expansion'
};
```
fix
```typescript
export enum ClipboardCopyVariant {
  inline = 'inline',
  expansion = 'expansion'
}
```
- onChange is not used  in ClipboardCopy.js
- copyButtonIdPrefix is not used in ClipboardCopy.js
- onCopy method isn't same type

```bash
(10,18): Interface 'ClipboardCopyProps' incorrectly extends interface 'HTMLProps<HTMLDivElement>'.
  Types of property 'onCopy' are incompatible.
    Type '((event: MouseEvent<Element>, text?: string | undefined) => void) | undefined' is not assignable to type '((event: ClipboardEvent<HTMLDivElement>) => void) | undefined'.
      Type '(event: MouseEvent<Element>, text?: string | undefined) => void' is not assignable to type '(event: ClipboardEvent<HTMLDivElement>) => void'.
        Types of parameters 'event' and 'event' are incompatible.
          Type 'ClipboardEvent<HTMLDivElement>' is missing the following properties from type 'MouseEvent<Element>': altKey, button, buttons, clientX, and 10 more.

```

**Comment by aljesusg** [comment]
cc @LHinson

**Comment by tlabaj** [comment]
closed by #1876
